src/react-medium-image-zoom.js

```diff
@@ -157,6 +159,7 @@ export default class ImageZoom extends Component {
   handleZoom(event) {
     if (this.props.shouldHandleZoom(event)) {
       this.setState({ isZoomed: true })
+      this.props.onZoom()
```

Comment:
It seems safer to me to pass this as a callback to `setState` so that it is triggered _after_ `setState` is completed and the component is re-rendered. Maybe something like
```js
this.setState({ isZoomed: true }, this.props.onZoom)
```
and the same for `onUnzoom`, but that could probably live inside of the `removeZoomed` method, instead?